### PR TITLE
[WIP] Kubernetes GlusterFS solution;

### PR DESCRIPTION
**Pull request by elijah-k-nebius**

### CODE REVIEW

File: kubernetes/terraform/kubernetes-glusterfs/variables.tf
Comment (malibora) [inline kubernetes/terraform/kubernetes-glusterfs/variables.tf:1]: disk type should also be configurable 

gpu nodes count?

Pls remove "Amount of SLurm nodes" comment

File: kubernetes/terraform/kubernetes-glusterfs/main.tf
[the remark this answers is not in the document]
Comment (elijah-k-nebius) [reply kubernetes/terraform/kubernetes-glusterfs/main.tf:1]: We've decided to solve this concern in a separate task. Leaving everything "as is" for now.

File: kubernetes/terraform/kubernetes-glusterfs/helm/mount-filesystem/templates/daemonset.yaml
[the remark this answers is not in the document]
Comment (elijah-k-nebius) [reply kubernetes/terraform/kubernetes-glusterfs/helm/mount-filesystem/templates/daemonset.yaml:1]: Done

File: kubernetes/terraform/kubernetes-glusterfs/variables.tf
Comment (elijah-k-nebius) [reply kubernetes/terraform/kubernetes-glusterfs/variables.tf:1]: Done

**Review by malibora — APPROVED** [review]

**Review by malibora — APPROVED** [review]

**Pull request merged**